prac_09/taxi_simulator.py

```diff
@@ -0,0 +1,72 @@
+"""Taxi Simulator"""
+
+from prac_09.taxi import Taxi
+from prac_09.silver_service_taxi import SilverServiceTaxi
+
+MENU = "q)uit, c)hoose taxi, d)rive"
+
+
+def main():
+    """Taxi Simulator program."""
+    taxis = [Taxi("Prius", 100), SilverServiceTaxi("Limo", 100, 2),
+             SilverServiceTaxi("Hummer", 200, 4)]
+    current_taxi = None
+    total_bill = 0
+
+    print("Let's drive!")
+    print(MENU)
+    menu_choice = input(">>> ").lower()
+
+    while menu_choice != "q":
+
+        if menu_choice == "c":
```

Comment:
For example the choose option could be its own function

Reply:
Hmmm, I was unsure due to the need for arguments to pass in and values to be returned, and how they work with classes/objects